### PR TITLE
Editable 24

### PR DESCRIPTION
**Pull request by Arthur-Milchior**
This is almost the same PR as https://github.com/hssm/advanced-browser/pull/69 , except that it's rebased on top of the recent changes, so it can work with .24
I kept two separate PR so that if you want, you can upload a version with my features for older version of anki.
Or you can close the other PR and just keep this one, this way it's easier, and anyway, it'll work for everyone as soon as they update

### CODE REVIEW

**Comment by hssm** [comment]
Thanks Arthur. I'll close the other PR. I don't like to maintain multiple versions.

I really like the idea of copying data from the table. Many times I've wished I could copy the card/note data IDs, so this will be really helpful.

Not sure how I feel about editable data though. Aside from being risky, there's also the HTML content to worry about. Also if you're clicking on the row to edit it then you open the note in the editor anyway. I'm not convinced it's worth all that extra complexity being added.

I'll have a play around with this tomorrow and see what it's like.

**Comment by Arthur-Milchior** [comment]
I totally understand your point about maintaining multiple version. However, I didn't know you enough to know what you think about it. I know you have two wonderful add-ons I love and that's mostly it.
Since Glutanimate usually tries to keep code compatible with multiple version, I thought it might be useful if you were like him

**Comment by Arthur-Milchior** [comment]
One of the point of having multiple commit is that you can take whatever you like. So you can merge the first commit and put it on ankiweb, and take time to consider the second one.
I decided that by default you can't edit unless you change the configuration. And even then, you get warning when important things are changed. So I tried to remain on the safe side with this PR.

**Comment by Arthur-Milchior** [comment]
Personally, I believe that the fact that the note is loaded should not really be taken into consideration. Indeed, when the note is loaded, you need to find the place where the field you want to edit is. Or you need to move your mouse down to the tag list. When I want to do a lot of quick editing, having a quick access to every lines with a single quick is extremely practical.
I'm not saying that it has already saved me enough time to justify the time it tooks to code this PR. However, it's certainly going to make the process of editing some values less costly to me

File: advancedbrowser/advancedbrowser/consts.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+"""Some values that are in anki/consts (in master) but not yet in 2.1.21"""
```
Comment (hssm) [inline advancedbrowser/advancedbrowser/consts.py]: I'm importing the constants now that we're targeting the next release for 2.1.24, so there's no need for this.

**Comment by hssm** [comment]
This also needs to be rebased.

**Comment by Arthur-Milchior** [comment]
Rebased

File: advancedbrowser/advancedbrowser/consts.py (hunk 1 above)
Comment (Arthur-Milchior) [reply advancedbrowser/advancedbrowser/consts.py]: I didn't expect 22 and 23 to come out so quickly when I made this PR. I admit this became useles. But honestly, it took me more than a year before Damien accepts my PR adding those constants in anki

**Pull request merged**